odoo-openmrs/src/test/java/com/ozonehis/eip/odoo/openmrs/client/OdooUtilsTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2021, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.eip.odoo.openmrs.client;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Avoid wildcards imports

Reply:
Thanks for pointing out. Something weird with my IDE it doesn't save this setting.